### PR TITLE
push multiple docker images with `docker push`

### PR DESCRIPTION
**Pull request by glyif**
**- What I did**
I changed `docker push` to be able to push multiple images.

feature request for: https://github.com/docker/cli/issues/267

**- How I did it**
I created a new function that looped runPush with the variable arguments passed into `docker push`. Changed `ExactArgs` to `RequireMinArgs` to support multiple images.

**- How to verify it**
You can run something like:
```
docker push glyif/valid_image:latest glyif/invalid_image:latest glyif/another_valid_image:latest
```

It will push the 2 valid images and return the error that glyif/invlid_image was unable to be pushed.

**- Description for the changelog**
ability to push multiple docker images with `docker push`

**- A picture of a cute animal (not mandatory but encouraged)**
My dogs again :)

### CODE REVIEW

File: cli/command/image/push.go [hunk 1]
```diff
@@ -29,7 +33,27 @@ func NewPushCommand(dockerCli command.Cli) *cobra.Command {
 	return cmd
 }
 
+func pushImages(dockerCli command.Cli, args []string) error {
+	var errs []string
+
+	for _, remote := range args {
+		if err := runPush(dockerCli, remote); err != nil {
```
Comment (glyif) [inline cli/command/image/push.go]: I'll make that change today :)

**Comment by aaronlehmann** [comment]
ping @stevvooe

**Comment by vdemeester** [comment]
@stevvooe I searched for it and just found https://github.com/moby/moby/issues/16106 (which is weird, I definitely remember other discussions about it 😓)

**Comment by thaJeztah** [comment]
Other discussions are in https://github.com/moby/moby/issues/7336, https://github.com/moby/moby/pull/11682, and related, https://github.com/moby/moby/issues/23383

**Comment by vdemeester** [comment]
As this PR is a bit stale and @ntindall opened a new one (https://github.com/docker/cli/pull/1021), I'm gonna close this one. Ping me (here) if you think we should re-open :wink:

**Pull request closed without merging**